packages/chart/src/components/BarChart/components/BarChart.Horizontal.tsx

```diff
@@ -75,7 +75,7 @@ export const BarChartHorizontal = () => {
                   const barDefaultLabel = !config.yAxis.displayNumbersOnBar ? '' : yAxisValue
 
                   // check if bar text/value string fits into  each bars.
-                  const textWidth = (getTextWidth as any)(xAxisValue, `normal ${fontSize[config.fontSize]}px sans-serif`)
+                  const textWidth = (getTextWidth as any)(barDefaultLabel, `normal ${fontSize[config.fontSize]}px sans-serif`)
```

Comment:
Modify the TS function in `packages/chart/src/types/ChartContext.ts` to support paramaters so you don't have to use this syntax or the any keyword. After modifying the syntax you should be able to use getTextWidth(string, string) when calling the function.

Reply:
It is modified but still gives red TS error. We need to update TS config main file but i did not do there anything in order not to mess things with configs